corehq/apps/reports/views.py

```diff
@@ -554,6 +554,11 @@ def user(self):
         return self.request.couch_user
 
 
+@dataclass
+class Timezone:
+    hours: str = "+00"
+    minutes: str = "00"
+
```

Comment:
if you add a `__str__` method on here then you wouldn't need to re-format in places that just want the display value
```suggestion

    def __str__(self):
        return f"{self.hours}:{self.minutes}"
```

If you convert the hours / minutes to int then:
```suggestion

    def __str__(self):
        return f"{self.hours:+03}:{self.minutes:02}"
```

```
for hours, minutes in [(0,0), (-1, 1), (-10, 10), (10, 10)]:
    print(f"{hours:+03}:{minutes:02}")
    
+00:00
-01:01
-10:10
+10:10
```